%\VignetteIndexEntry{FUCHIKOMA}
\documentclass[11pt]{article}
\usepackage{Sweave}
\usepackage{amsmath}
\usepackage{hyperref}
\usepackage{cite}
\usepackage[numbers]{natbib}
\usepackage{amssymb}
% keep.source: T => With comment out line in each chunk
\input{FUCHIKOMA-concordance}
\usepackage{url}
\usepackage[utf8]{inputenc}

\setlength{\textheight}{8.5in}
\setlength{\textwidth}{6in}
\setlength{\topmargin}{-0.25in}
\setlength{\oddsidemargin}{0.25in}
\setlength{\evensidemargin}{0.25in}
\newcommand{\Rpackage}[1]{{\textit{#1}}}

\usepackage{Sweave}
\begin{document}

\title{\bf Fuchikoma: Revealing differentially expressed genes with non-linear dimensional reduction using single cell RNA-Seq data}
\author{Koki Tsuyuzaki$^1$, Haruka Ozaki$^1$, Mika Yoshimura$^1$, and Itoshi Nikaido$^1$.}
\maketitle
\begin{center}
\noindent
$^1$Bioinformatics Research Unit, Advanced Center for Computing and Communication, RIKEN.\\
\end{center}

\begin{center}
{\tt k.t.the-answer@hotmail.co.jp}
\end{center}
\tableofcontents

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%\clearpage
\newpage
\section{Introduction}
%This document provides the way to perform meta-analysis of RNA-seq data using \Rpackage{metaSeq} package \cite{NOISeq}.

%\begin{center}
%  \begin{table}[htbp]
%    \begin{tabular}{|c|c|c|}\hline
%      ID in this vignette & Accession (SRA / ERA Accession) & Experimental Design \\ %\hline \hline
%      StudyA & SRP008746 & Breast Cancer (n=3) vs Normal (n=2) \\ \hline
%      StudyB & SRP006726 & Breast Cancer (n=1) vs Normal (n=1) \\ \hline
%      StudyC & SRP005601 & Breast Cancer (n=7) vs Normal (n=1) \\ \hline
%      StudyD & ERP000992 & Breast Cancer (n=2) vs Normal (n=1) \\ \hline
%\end{tabular}
%  \end{table}
%\end{center}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%\begin{figure}[ht]
%\centering
%\includegraphics[width=80mm]{Fig1.jpeg}
%\caption{Difference of the number of reads}
%\label{fig1}
%\end{figure}
%\newpage
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\section{Getting started}
%At first, install and load the \Rpackage{metaSeq} and \Rpackage{snow}.
%\begin{center}
%<<echo=TRUE,print=FALSE>>=
%library("metaSeq")
%library("snow")
%@
%\end{center}


\section{Setup}
This vignette was built on:
\begin{Schunk}
\begin{Sinput}
> sessionInfo()
\end{Sinput}
\begin{Soutput}
R version 3.2.2 (2015-08-14)
Platform: x86_64-apple-darwin13.4.0 (64-bit)
Running under: OS X 10.11.1 (El Capitan)

locale:
[1] ja_JP.UTF-8/ja_JP.UTF-8/ja_JP.UTF-8/C/ja_JP.UTF-8/ja_JP.UTF-8

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods  
[7] base     

loaded via a namespace (and not attached):
[1] tools_3.2.2
\end{Soutput}
\end{Schunk}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\newpage

\vspace{2cm}

\begin{thebibliography}{9}
% \providecommand{\natexlab}[1]{#1}
% \providecommand{\url}[1]{\texttt{#1}}
% \expandafter\ifx\csname urlstyle\endcsname\relax
%   \providecommand{\doi}[1]{doi: #1}\else
%   \providecommand{\doi}{doi: \begingroup \urlstyle{rm}\Url}\fi

\bibitem{NOISeq}
Tarazona, S. and Garcia-Alcalde, F. and Dopazo, J. and Ferrer, A. and Conesa, A.
\newblock {Genome Research}
\newblock \emph {Differential expression in RNA-seq: A matter of depth}, 21(12): 2213-2223, 2011.

\end{thebibliography}

\end{document}
